app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,25 @@
+class FoodsController < ApplicationController
```

Comment:
Kindly add the index action to the FoodsController as this was required and add a food index view the display the list of foods 
check the list of requirements:
![Screenshot from 2024-02-10 22-17-10](https://github.com/BiancaMe/recipe-app/assets/59707859/cc988b76-80c8-4f01-ac0f-e918d8bbbeaf)

Reply:
- Hello @souad988, after having discussions with a SCR, I have been informed to overlook this requirement since they started off as a team of 3 and implemented the requirements

Reply:
I actually was pointing to food list which is required for a team that has fewer than 3 members

Reply:
- Good to go team 👍🏽